src/main/java/frc/robot/Robot.java

```diff
@@ -88,12 +86,11 @@ public void robotInit() {
         SmartDashboard.putData("unloadBallToShooter", indexer.unloadBall());
         SmartDashboard.putData("ShootNoSpinup", indexer.shootBall());
         SmartDashboard.putData("index ball", indexer.indexBall());
-        SmartDashboard.putData("lift brake toggle", lift.toggleBrake());
         SmartDashboard.putData("Deploy intake", intake.deployIntake());
         SmartDashboard.putData("Retract intake", intake.retractIntake());
 
         SmartDashboard.putData("cam toggle", camToggle());
-        SmartDashboard.putData("After Match Lift Sequence", afterMatchPit());
+        SmartDashboard.putData("After Match Retract Intake", intake.retractIntake());
```

Comment:
Do we need this? I'm pretty sure it defaults to retracting.

Reply:
actually yeah it does